docs/references/authentication_oidc.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_label: "Authentication"
+title: ""
+---
+
+# Authentication
+
+Since version **1.3.0** Epinio has integrated [Dex](https://dexidp.io/) as identity provider, adding supports for external OIDC providers.
```

Comment:
```suggestion
Since version **1.3.0**, Epinio has integrated [Dex](https://dexidp.io/) as an identity provider which adds the support for external OIDC providers.
```